bbcode-src/preset.js

```diff
@@ -6,16 +6,10 @@ import { check } from "./tags/check";
 import { color } from "./tags/color";
 import { divide } from "./tags/divide";
 import { font } from "./tags/font";
-import { highlight } from "./tags/highlight";
```

Comment:
Removing these lines will cause the imports for these tags to fail. I don't see them elsewhere in the file you're checking in. They should be added back in.

 If you don't have your local branch set to track the remote/main branch, you won't get any of the changes we push up. You can set your local branch to track using `git branch -u origin/main`. Then you can fetch and pull and you should see the new changes. If you run a `git log` the last commit should be this one: https://github.com/RpNation/bbcode/commit/2210763df5973037ee378f399179afa4c05ce48c

Reply:
The lines have been re-added and I can confirm that I'm indeed seeing that commit as the latest one!